autonomous_software_pkg/src/keyboard_teleop.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/python3
+
+
+import rospy
+from std_msgs.msg import String
+from pynput.keyboard import Key, Listener
+
+
+class TeleopPublisher:
+    def __init__(self):
+        rospy.init_node('keyboard_teleop_controller')
+        self.speed = 100.0
+        self.pub = rospy.Publisher('teleop_speed', String, queue_size=10)
```

Comment:
In the controller, line 131 says that the topic is of type int, but here it says string. Best is probably to set it to int everywhere.

Reply:
Great point, thank you.

Changed to Int32